simmechanics/data/icub3/ICUB_3_joint_all_parameters.csv

```diff
@@ -3,10 +3,10 @@ torso_yaw,-43,43,2,0,120,
 torso_pitch,-18,45,2,0,120,
 torso_roll,-23,23,2,0,120,
 r_shoulder_pitch,-88,13,2,0,120,
-r_shoulder_roll,12,160,2,0,120,
+r_shoulder_roll,12,163,2,0,120,

```

Comment:
Did we documented somewhere this limits? It would be great if we could have some official documentation, such as https://icub-tech-iit.github.io/documentation/icub_kinematics/icub-joints-limits/icub-joints-limits/ , so that here we just reference those.

Reply:
I don't think we did yet, since there was still going to be an update of the limits for the real robot, but we can update them I think

Reply:
If they are not updated, I am ok in merging this PR before the documentation changed, but I would open an issue on the update of joint limits, otherwise we may forget about it.

Reply:
Regarding this point I think we need to discuss with @pattacini about these limits. Were they set for iCub3 anywhere, since the joint calibration? Where should we get these values from?

Reply:
@traversaro's suggestions were:
1. to update our official documentation website with the iCub 3 joint limits.
2. to create in robots-configuration an issue in order to keep track of the need for checking these new limits.

I think you can go ahead certainly with 2; it'd be great to have also 1 done.

Reply:
I have opened a new issue in robots-configuration, that we can see here: https://github.com/robotology/robots-configuration/issues/280

this should allow us to keep track of this, in order to update them in the future

Reply:
Thanks!